cfme/tests/services/test_rest_services.py

```diff
@@ -778,3 +778,77 @@ def test_edit_orchestration_templates(self, rest_api, orchestration_templates, f
             assert edited[i].description == new[i]['description']
             orchestration_templates[i].reload()
             assert orchestration_templates[i].description == new[i]['description']
+
+    @pytest.mark.tier(3)
+    @pytest.mark.uncollectif(lambda: version.current_version() < '5.8')
+    @pytest.mark.parametrize(
+        "from_detail", [True, False],
+        ids=["from_detail", "from_collection"])
+    def test_copy_orchestration_templates(self, request, rest_api, orchestration_templates,
+            from_detail):
+        """Tests copying of orchestration templates.
+
+        Metadata:
+            test_flag: rest
+        """
+        response_len = len(orchestration_templates)
+        new = []
+        for _ in range(response_len):
+            uniq = fauxfactory.gen_alphanumeric(5)
+            new.append({
+                "name": "test_copied_{}".format(uniq),
+                "content": "{ 'Description' : " + "'{}'".format(uniq) + "}\n"
```

Comment:
Not really valid JSON, more like raw string containing JSON data (see '\n' at the end). There would need to be string concatenation in any case. I'll try to use ``replace`` to make it less cryptic.